### PR TITLE
parser for ncov-gambia

### PR DESCRIPTION
**Pull request by ekamau**
First draft for the ncov-Gambia data.

### CODE REVIEW

**Comment by sadiekelly** [comment]
@pipliggins @ekamau please could someone point me in the direction of where the inability to walk scale attribute is defined, with the response options for each coding. I just want to make sure that those in ncov gambia match up with the options defined previously but can't find the documentation. Thanks!

**Comment by pipliggins** [comment]
> @pipliggins @ekamau please could someone point me in the direction of where the inability to walk scale attribute is defined, with the response options for each coding. I just want to make sure that those in ncov gambia match up with the options defined previously but can't find the documentation. Thanks!

https://github.com/globaldothealth/isaric/blob/main/docs/guide.rst#inability-to-walk-scale

File: isaric/parsers/ncov_gambia_parser.toml
Comment (sadiekelly) [inline isaric/parsers/ncov_gambia_parser.toml:1]: @ekamau here's my review of the gambia parser - let me know if you want to discuss anything further - thanks! 
[Gambia CVRVRNJ parser review 2023-10-06.xlsx](https://github.com/globaldothealth/isaric/files/12831174/Gambia.CVRVRNJ.parser.review.2023-10-06.xlsx)

**Comment by ekamau** [comment]
@pipliggins - I have reviewed the changes and just added context for temperature observations. all look good now.
I am happy for you to merge. thank you!

**Pull request merged**